tools/encyclopedia/encyclopedia_encyclopedia.xml

```diff
@@ -0,0 +1,146 @@
+<tool id="encyclopedia_encyclopedia" name="EncyclopeDIA" version="@VERSION@.0">
+    <description>Library Searching Directly from Data-Independent Acquisition (DIA) MS/MS Data</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <command detect_errors="aggressive"><![CDATA[
+        @CMD_IMPORTS@
+        @LINK_SCAN_INPUT@
+        @LINK_FASTA_INPUT@
+        @LINK_LIB_INPUT@
+        EncyclopeDIA -Djava.awt.headless=true -Xmx\$[ \${GALAXY_MEMORY_MB:-20480} / 1024 ]g
+        @SCAN_INPUT@
+        @FASTA_INPUT@
+        @LIB_INPUT@
+        @COMMON_OPTIONS@
+        @MASS_LIBRARY_TOLERANCE@
+        @PERCOLATOR_OPTIONS@
+        @PEAK_OPTIONS@
+        @WINDOW_OPTIONS@
+        @MODIFICATION_OPTIONS@
+        -o gxpedia
+    ]]></command>
+    <inputs>
+        <expand macro="scan_input"/>
+        <expand macro="fasta_input"/>
+        <expand macro="lib_input" optional="false" libhelp="Chromatogram elib from SearchToLib on pooled samples"/>
+        <expand macro="options_section"/>
+        <param name="select_outputs" type="select" label="Select outputs" multiple="true">
+            <option value="log" selected="true">log</option>
+            <option value="elib" selected="true">elib</option>
+            <option value="first_delta_rt_pdf" selected="false">encyclopedia.txt.first.delta_rt.pdf</option>
```

Comment:
should we add here a proper text and not strange file name?

Reply:
Yes.  I'll do that.